curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5e54e3a154c8078ed48.md

```diff
@@ -86,15 +119,31 @@ Unfortunately, we've just realized we read the requirements wrong. We need to al
 const regex = /^[a-zA-Z]{0,}\d{4,6}$/;
 ```
 
-But our pattern is getting long again. Thankfully, there's another short-hand for "match the previous character zero or more times" –  the asterisk symbol. Let's replace our quantifier with that in the pattern, and test it:
+But our pattern is getting long again. Thankfully, there's another short-hand for "match the previous character zero or more times" –  the asterisk (`*`) symbol. Let's replace our quantifier with that in the pattern, and test it:
 
-![image](https://cdn.freecodecamp.org/curriculum/lecture-transcripts/what-are-regex-quantifiers-and-how-do-they-work-5.png)
+```js
+const regex = /^[a-zA-Z]*\d{4,6}$/;
+console.log(regex.test("123")); // false
+console.log(regex.test("a1234")); // true
+console.log(regex.test("12345")); // true
+console.log(regex.test("az12345")); // true
+console.log(regex.test("X123456")); // true
+console.log(regex.test("1234567")); // false
+```
 
 Now we successfully match any identifier with zero or more letters followed by four to six numbers. But it turns out this is crashing our system – we actually have to require at least one letter.
 
-Again, we could use a quantifier with a minimum of one and no defined maximum, or we could use yet another special syntax – the plus symbol:
+Again, we could use a quantifier with a minimum of one and no defined maximum, or we could use yet another special syntax – the plus (`+`) symbol:
```

Comment:
```suggestion
Again, we could use a quantifier with a minimum of one and no defined maximum, or we could use yet another special syntax – the plus symbol (`+`):
```
Same as before

Reply:
I have seen it both ways. So it should be fine to leave it as is